template.yaml

```diff
@@ -140,6 +140,9 @@ Resources:
         RestrictPublicBuckets: true
       VersioningConfiguration:
         Status: Enabled
+      Tags:
+        - Key: SensitiveData
```

Comment:
I'd say we probably want the tag key to be PII or PersonalIdentifyingInformation - there is actually a bit of a difference between PII and sensitive data - 

> PII is the most commonly available and least regulated type of data, and may or may not be sensitive — or may be considered sensitive only under certain circumstances, or when combined with other data about an individual.

>For example, PII like names, phone numbers, or other information that may be widely publicly available, is not usually considered sensitive (though could be in certain contexts), whereas PII like social security numbers, alien registration numbers, or driver’s license numbers would always be sensitive.

from https://bigid.com/blog/sensitive-information-guide/